lib/checks/color/color-contrast.js

```diff
@@ -1,6 +1,12 @@
+var useScroll = true;
+
+if((options || []).indexOf('noScroll') !== -1) {
```

Comment:
The options can be anything you specify. In this case, I think it would be better to make it an object with an attribute that is a boolean (truthy test), I would suggest changing this to:

```
if (!(options || {}).noScroll) {
```

and making the appropriate changes to the options below